### PR TITLE
Update cross-tenant-sharepoint-migration-step6.md

### LINKED ISSUE #12133: Incorrect Example - Start a SharePoint Cross-tenant site migration
I believe wrong cmdlet example is listed here, should show cmdlet for moving a site but shows cmdlet for moving a group connected site - https://learn.microsoft.com/en-us/microsoft-365/enterprise/cross-tenant-sharepoint-migration-step6?view=o365-worldwide#start-a-sharepoint-cross-tenant-site-migration

Example given: Start-SPOCrossTenantGroupContentMove  -SourceGroupAlias <…> -TargetGroupAlias <…> -TargetCrossTenantHostUrl <…>

**Example should be:  Start-SPOCrossTenantSiteContentMove -SourceSiteUrl <string> -TargetSiteUr] <string> -TargetCrossTenantHostUrl <string>**

image

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: ecdf32bb-048f-e9ea-34b2-20e810ba0776
* Version Independent ID: ecdf32bb-048f-e9ea-34b2-20e810ba0776
* Content: [SharePoint site Cross-tenant SharePoint migration Step 6 (preview) - Microsoft 365 Enterprise](https://learn.microsoft.com/en-us/microsoft-365/enterprise/cross-tenant-sharepoint-migration-step6?view=o365-worldwide)
* Content Source: [microsoft-365/enterprise/cross-tenant-sharepoint-migration-step6.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/enterprise/cross-tenant-sharepoint-migration-step6.md)
* Service: **sharepoint-online**
* Sub-service: **sharepoint-migration**
* GitHub Login: @JoanneHendrickson
* Microsoft Alias: **jhendr**

### PR DESCRIPTION
**Pull request by msbemba**
Made changes to the parameters in the example

Fixes https://github.com/MicrosoftDocs/microsoft-365-docs/issues/12133

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/enterprise/cross-tenant-sharepoint-migration-step6.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ Now you're ready to start your SharePoint migration. Before starting any cross-t
 2. To start the migration, a SharePoint Online Admin or Microsoft 365 Global Admin of the source tenant must run the following command:
 
 ```PowerShell
-Start-SPOCrossTenantGroupContentMove  -SourceGroupAlias <…> -TargetGroupAlias <…> -TargetCrossTenantHostUrl <…>
+Start-SPOCrossTenantGroupContentMove  -SourceSiteUr <…> -TargetSiteUrl <…> -TargetCrossTenantHostUrl| <…>
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/cross-tenant-sharepoint-migration-step6.md]:
```suggestion
Start-SPOCrossTenantGroupContentMove  -SourceSiteUrl <…> -TargetSiteUrl <…> -TargetCrossTenantHostUrl| <…>
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@JoanneHendrickson Please merge this PR. Thanks

**Comment by JoanneHendrickson** [comment]
Having this confirmed by the cross-tenant product group.

**Review by JoanneHendrickson — APPROVED** [review]

**Comment by JoanneHendrickson** [comment]
Product team confirmed the change. thanks all!

**Pull request merged**